app/screens/TalkDetailsScreen/AssistantsList.tsx

```diff
@@ -17,15 +16,13 @@ export function AssistantsList(props: AssistantsListProp) {
 
   return (
     <>
-      <View style={$assistantContainer}>
+      <View style={$assistantRoot}>
```

Comment:
Last minor detail. If we have a root view, do we need the containing `<></>`?

Reply:
good eye. fixed here: https://github.com/infinitered/ChainReactApp2023/pull/166/commits/f06e85655e44e0efd1f8fd0db04f18eebbca77dd